### DISCUSSION TITLE
Tuna OBS Plug-in Integration

### DISCUSSION THREAD

**Discussion by trevorraney** [pear-devs/pear-desktop#436] [Q&A]
A bit new with the Tuna OBS plug-in and do not intend to make this a support discussion for *that*. But it's not clear to me how this YouTube Music app integrates with Tuna.

It is enabled in the settings, and confirmed by looking at config.json.

I tried just setting up the webserver with the default port, as listed in [mesmerx's comment on #372](https://github.com/th-ch/youtube-music/issues/372#issuecomment-916304894), but the log states connection is timing out.

My next assumption is that YouTube Music is creating .txt files with music information, and .png files for cover art, but I don't think that's the case. 

Would appreciate guidance from anyone willing to provide it!

**Comment by Araxeus** [comment]
It should post info on webserver `http://localhost:1608`

Did you specify song source as web browser?
![image](https://user-images.githubusercontent.com/78568641/137546888-141f72b7-6ecd-4261-bf66-9f789fb8d7d3.png)
